### ISSUE TITLE
Service Fabric Activation Error 

### ISSUE THREAD

**Issue by TLena** [Azure/service-fabric-issues#655]
Hello, 

During launching of Service Farbric Application I got the error
> Error event: SourceId='System.Hosting', Property='CodePackageActivation:Code:EntryPoint'.
There was an error during CodePackage activation.The service host terminated with exit code:2147516556

Team, could you be so kind to provide any details regarding 2147516556? What exactly this status code means? 

Thank you in advance.

**Comment by vturecek** [comment]
What are you running in your application? That exit code is from your service process so it's hard to tell.. just searching around a bit I don't see much for that error code.

**Comment by masnider** [comment]
@TLena yeah that error code doesn't look like one that's coming from inside Service Fabric or the OS. Can you instrument your code and try to capture the stack trace where you're running into this?
